custom_components/renault/sensor.py

```diff
@@ -246,7 +247,7 @@ def unit_of_measurement(self):
         return LENGTH_KILOMETERS
 
 
-class RenaultRangeSensor(RenaultBatteryDataEntity):
+class RenaultRangeBatterySensor(RenaultBatteryDataEntity):
     """Range sensor."""
```

Comment:
Update the description.
```suggestion
    """Battery Autonomy sensor."""
```